src/AzSK.ADO/SVT/SVT.ps1

```diff
@@ -249,7 +249,13 @@ function Get-AzSKADOSecurityStatus
 		[switch]
 		[Parameter(HelpMessage="Skip organization and user controls.")]
 		[Alias("souc")]
-		$SkipOrgUserControls
+		$SkipOrgUserControls,
+
+		[string]
+		[Parameter(Mandatory = $false, HelpMessage="Name of the repo to fetch org policy.")]
+		[ValidateNotNullOrEmpty()]
+		[Alias("rp")]
+		$RepoName
```

Comment:
use PolicyRepoName as it can overload with Repo SVT in future

Reply:
Changed as suggested